libs/parallelism/algorithms/include/hpx/parallel/algorithms/rotate.hpp

```diff
@@ -97,55 +264,16 @@ namespace hpx { namespace parallel { inline namespace v1 {
         /// \endcond
     }    // namespace detail
 
-    /// Performs a left rotation on a range of elements. Specifically,
-    /// \a rotate swaps the elements in the range [first, last) in such a way
-    /// that the element new_first becomes the first element of the new range
-    /// and new_first - 1 becomes the last element.
-    ///
-    /// \note   Complexity: Linear in the distance between \a first and \a last.
-    ///
-    /// \tparam ExPolicy    The type of the execution policy to use (deduced).
-    ///                     It describes the manner in which the execution
-    ///                     of the algorithm may be parallelized and the manner
-    ///                     in which it executes the assignments.
-    /// \tparam FwdIter     The type of the source iterators used (deduced).
-    ///                     This iterator type must meet the requirements of an
-    ///                     forward iterator.
-    ///
-    /// \param policy       The execution policy to use for the scheduling of
-    ///                     the iterations.
-    /// \param first        Refers to the beginning of the sequence of elements
-    ///                     the algorithm will be applied to.
-    /// \param new_first    Refers to the element that should appear at the
-    ///                     beginning of the rotated range.
-    /// \param last         Refers to the end of the sequence of elements the
-    ///                     algorithm will be applied to.
-    ///
-    /// The assignments in the parallel \a rotate algorithm invoked
-    /// with an execution policy object of type \a sequenced_policy
-    /// execute in sequential order in the calling thread.
-    ///
-    /// The assignments in the parallel \a rotate algorithm invoked with
-    /// an execution policy object of type \a parallel_policy or
-    /// \a parallel_task_policy are permitted to execute in an unordered
-    /// fashion in unspecified threads, and indeterminately sequenced
-    /// within each thread.
-    ///
-    /// \note The type of dereferenced \a FwdIter must meet the requirements
-    ///       of \a MoveAssignable and \a MoveConstructible.
-    ///
-    /// \returns  The \a rotate algorithm returns a
-    ///           \a hpx::future<tagged_pair<tag::begin(FwdIter), tag::end(FwdIter)> >
-    ///           if the execution policy is of type
-    ///           \a parallel_task_policy and
-    ///           returns \a tagged_pair<tag::begin(FwdIter), tag::end(FwdIter)>
-    ///           otherwise.
-    ///           The \a rotate algorithm returns the iterator equal to
-    ///           pair(first + (last - new_first), last).
-    ///
+
+    //clang-format off    
```

Comment:
nitpick: could you write this as `// clang-format off` (i.e. add a space after the `//`)